components/ArticleReplyFeedbackControl/ArticleReplyFeedbackControl.js

```diff
@@ -136,6 +136,13 @@ function ArticleReplyFeedbackControl({ articleReply, className }) {
   const openVotePopover = (event, value) => {
     setVotePopoverAnchorEl(event.currentTarget);
     setVote(value);
+    createReplyFeedback({
+      variables: {
+        articleId: articleReply.articleId,
+        replyId: articleReply.replyId,
+        vote: value,
```

Comment:
This will cause issue in the following user sequence:
1. The user clicks upvote, fills in some reason and then send
2. The user clicks upvote again to open popup
3. The user tries to dismiss the popup

We expect that the user's reason in step 1 should be preserved. However, the current implementation will wipe the reason entirely.

We may argue that the existing feedback button does not load the reason either. However, in this PR, it makes it easier for the user to clear the reason by accident.

I suggest we try amending the reason loading logic before accepting this change.

Reply:
So, if I understand correctly, we want to preserve the user submitted reason when they open the popup?
And optionally create a reply feedback on upvote / downvote with the existing reason if it exists?